web/gui/dashboard_info.js

```diff
@@ -1507,6 +1507,10 @@ netdataDashboard.context = {
         height: 0.5,
         info: 'The average service time for completed I/O operations. This metric is calculated using the total busy time of the disk and the number of completed operations. If the disk is able to execute multiple parallel operations the reporting average service time will be misleading.'
     },
+    'disk.latency_io': {
+        height: 0.5,
+        info: 'Latency Output is the time it takes for an Output request to be completed. Latency is the single most important metric to focus on when it comes to storage performance, under most circumstances. For hard drives, an average latency somewhere between 10 to 20 ms can be considered acceptable. For SSD (Solid State Drives), depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than 1ms latency numbers. The dimensions refer to time intervals. This chart is based on the <a href="https://github.com/cloudflare/ebpf_exporter/blob/master/examples/bio-tracepoints.yaml" target="_blank">bio_tracepoints</a> tool of the ebpf_exporter.'
```

Comment:
`Latency Output` -> `I/O latency`,
`Output request` -> `I/O request`

Reply:
Thank you, fixed!